migration/src/test/java/org/axonframework/migration/AxonJakartaTest.java

```diff
@@ -0,0 +1,79 @@
+package org.axonframework.migration;
+
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.java.Assertions.mavenProject;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+public class AxonJakartaTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+                .parser(JavaParser.fromJavaVersion()
+                        .logCompilationWarningsAndErrors(true)
+                        .classpath("axon-messaging", "rewrite-spring"))
+                .recipe("/META-INF/rewrite/axon-jakarta.yml", "org.axonframework.axon.UpgradeAxonFramework_4_7_Jakarta");
+    }
+
+    @Test
+    void migrateImports() {
+        //language=java
+        rewriteRun(
+                java(
+                        "package sample.axon;\n" +
```

Comment:
Curious: I assume the first `String` is intended to inject a sample java class while, where the second is the `expected` outcome after performing the rewrite?

Reply:
Yes! We call them `before` and `after`. It's a little better viewing this in an IDE rather than on GitHub.

Reply:
Awesome, thanks for the explanation.